site/zenodo_rdm/openaire/serializers/schema.py

```diff
@@ -196,10 +196,15 @@ def _reverse_funder_acronym(funder_ror):
                 funder_acronym = _reverse_funder_acronym(funder_ror)
                 award_program = award.get("program", "")
                 award_number = award.get("number", "")
+                award_title = award.get("title", {}).get("en")
+                award_acronym = award.get("acronym", "")
                 if funder_acronym and award_program and award_number:
-                    links.append(
-                        f"info:eu-repo/grantAgreement/{funder_acronym}/{award_program}/{award_number}"
-                    )
+                    b_link = f"info:eu-repo/grantAgreement/{funder_acronym}/{award_program}/{award_number}"
+                    if award_title:
+                        b_link += f"/{award_title}"
```

Comment:
major: title and acronyms might include slashes `/`. In legacy we [URL-encoded them](https://github.com/zenodo/zenodo/blob/482ee72ad501cbbd7f8ce8df9b393c130d1970f7/zenodo/modules/openaire/schema.py#L134). I would apply the same transformation all the fields (i.e. `funder_acronym`, `award_program`, `award_number`, `award_title`, `award_acronym`)

Reply:
I encoded them using [urllib.parse.quote_plus](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus) for complete URL encoding. I.e. spaces were not being URL encoded in legacy.